### ISSUE TITLE
Large-scale weakly-supervised pre-training for video action recognition

### ISSUE THREAD

**Issue by KangHoyong** [KangHoyong/AI_sub_projects#2]
# Large-scale weakly-supervised pre-training for video action recognition 리뷰
비디오 액션 인식을 위한 대규모 pre-training 

### Abstract 

This paper presents an in-depth study of using large volumes of web videos for per-training video models for the task of action recognition
본 논문에서는 action recognition 인식 과제를 위한 per-training video에 대량의 web videos 를 사용하는 방법에 대한 심층적인 연구를 제시

Further, we examine three questions in the construction of weakly-supervised video action datasets.
weakly-supervised video action datasets 구축에 있어 세 가지 문제를 검토해야한다.

1.  First, given that actions involve interactions with objects, how should one construct a verb-object pre- training label space to benefit transfer learning the most?
-> action에 는 대상과의 상호작용이 수반되는 점을 감안할 때, 어떻게 하면 이전 학습에 가장 유리하도록 verb-object pre- training 라벨 공간을 구축해야 하는가 ?

2. frame-based models perform quite well on action recognition; is pre-training for good image features sufficient or is pre-training for spatio-temporal features valu- able for optimal transfer learning? 
->  frame-based models 기반 동작인식에서 상당히 우수한 성능을 발휘한다. 
-> 고려해야 할 사항으로 좋은 이미지가 충분히 있는가 ? 또는 가치가 있는가 ?!

3. actions are generally less well-localized in long videos vs. short videos; since action labels are provided at a video level, how should one choose video clips for best performance, given some fixed budget of number or minutes of videos?
-> actions 일반적으로 long videos and short videos 에서 잘 표현되지 않는다 
-> action labels 은 비디오 레벨에서 제공되기 때문에 비디오 number or minutes 이라는 고정된 fixed budget 으로 볼 때 최고의 성능을 위해 비디오 클립을 어떻게 선택해야 하는가?! 

## Introduction 
최근의 연구 [A. Joulin, L. van der Maaten, A. Jabri, and N. Vasilache. Learning visual features from large weakly supervised data. ECCV, 2016, C. Sun, A. Shrivastava, S. Singh, and A. Gupta. Revisit- ing unreasonable effectiveness of data in deep learning era. ICCV, 2017 , D. Mahajan, R. Girshick, V. Ramanathan, K. He, M. Paluri, Y. Li, A. Bharambe, and L. van der Maaten. Exploring the limits of weakly supervised pretraining. ECCV, 2018]는 수억 개의 noisy web image 에 대한 per-training 이 significantly boots the state-of-the-art in object classification 를 현저하게 증가시킨다는 것을 보여주었다.

이러한 성공이 비디오로 이어지기를 바라는 사람은 분명 존재한다 하지만 비디오 action 인지도는 image tasks 없는 독특한 과제를 제시한다.

4가지 과제는 다음과 같이 제시된다 .
1. web image 는 주로 label noise ( missing or incorrect object labels) 이러한 문제에 직면한다 .
반면 wild videos 경우  2 가지 문제가 있다 (label noise, temporal noise due to the lack of localization of action labels. -> ??? ) 

In real-world videos, a given action typically occupies only a very small portion of a video. 
-> 일반적인 비디오 영상에서 주어진 동작은 아주 작은 부분을 차지한다.

2. Second, in prior work on images, labels were restricted to scenes and objects (i.e., nouns).
-> 이미지에 관한 선행 작업에서는 fables 장면이나 사물로 제한 하였다
하지만 action labels 은 일반적으로 최소 한개의 one verb-object pair 을 포함해야한다.

However, action labels (eg: “catching a fish”) are more complex, typically involv- ing at least one verb-object pair. Further, even at large scale, many valid verb-object pairs may be observed rarely or never at all; for example, “catching a bagel” is an entirely plausible action, but rarely observed. 
-> action labels 은 일반적으로 최소 한개의 one verb-object pair 을 포함해야하고 복잡하다  또한 유효한 많은 one verb-object pair 쌍은 거의 관찰되지 않거나 전혀 관찰되지 않을 수 있다. 예를 들어 "catching a bagel" 잡는 것은 전적으로 그럴듯한 행동이지만 관찰되지 않는다.

Therefore, it is natural to inquire: is it more useful to pre-train on labels chosen from marginal distributions of nouns and verbs, do we need to pre-train on the observed portion of the joint distribution of (verb, noun) labels, or do we need to focus entirely on the target dataset’s labels? 
-> 명사와 동사의 한계 분포에서 선택한 labels 을 pre-train으로 가치가 있는가 ? 아니면 (verb 동사, noun 명사) 라벨의 공동 분포에서 관찰된 부분을 사전 교육해야 하는가 ? 아니면 대상 data 집합의 labele 에 전적으로 초점을 맞춰야 하는가 ?!

3. In addition, longer videos contain more frames, but short videos presumably contain more relevant frames; what is the best choice of video lengths when con- structing a pre-training dataset?
-> 긴비디오는 더 많은 프레임을 포함하지만 짧은 비디오는 아마도 더 많은 관련 프레임을 포함하고 있을 것이다.

4. we question whether pre-training on videos (vs images) is even necessary.
우리는 비디오 ( vs images) 에 대한 사전 pre-training 이 필요한지에 대해 의문을 제기한다.

즉 논문에서는 비디오 클립에 대한 per-training 이 실제로 증가된 컴퓨팅의 가치가있는가 ? 아니면 Strong image 많으로 충분한가?! 
아주 자세하게 다룬다. 

논문의 핵심 목표는 모델 아키텍처 설계를 보완하는 train data 에만 집중함으로 써 학습된 비디오 기능 표현을 개선하는 것이다.
